<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sudoku</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color : #000000;
}</string>
   </property>
   <widget class="QLineEdit" name="mainLine">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>601</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color: #ffffff;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Sudoku Game</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Sudoku Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlay">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnPlay{
	color:#ffffff;
	border-style: solid;
	border-width: 5px;
	border-radius : 50%;
}</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEasy">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>268</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #ffffff;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Easy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNormal">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #ffffff;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHard">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #ffffff;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Hard</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
